README.md

```diff
@@ -28,6 +28,38 @@ OfficeDevPnP.PowerShell | Erwin van Hunen
 A build script will copy the required files to a folder in your users folder, called:
 *C:\Users\<YourUserName>\Documents\WindowsPowerShell\Modules\OfficeDevPnP.PowerShell.Commands*
 
+Alternatively for installation on machines that have at least PowerShell v3 installed (you can find this out by opening PowerShell and running $host.version and Major should be above 3) you can run the below command which will install PowerShell Package Management and then install the PowerShell Modules from the PowerShell Gallery
+```powershell
+Invoke-Expression (New-Object Net.WebClient).DownloadString('http://bit.ly/InstallO365PnPPowerShell')
+```
+
+If you wish to see the commands that the above will run please see the files as stored in the below locations
+http://bit.ly/PSPackManInstall
```

Comment:
Hi! Can you include those files in the repository instead? So we don't have a dependency to an external location. You can store them in a subfolder of Samples folder.